frontend/src/state/actions.ts

```diff
@@ -0,0 +1,12 @@
+import { action } from 'typesafe-actions';
+import {Person, Request, User} from './index';
+
+export const enum ActionTypes {
+    UPDATE_USER = 'UPDATE_USER',
+    UPDATE_PEOPLE = 'UPDATE_PEOPLE',
+    UPDATE_REQUESTS = 'UPDATE_REQUESTS'
+}
+
+export const updateUser = (user: User) => action(ActionTypes.UPDATE_USER, user);
+export const updatePeople = (...people: Array<Person>) => action(ActionTypes.UPDATE_PEOPLE, people);
```

Comment:
Why is this `updatePeople` and `updateRequests`, instead of `updatePerson` and/or `updateRequest`?
Wouldn't it be better to send a single updated person/request per action to prevent redundancy?